Update 01_table_material.sql

pinguim11 opened the pull request:
criação da tabela material

pinguim11 committed:
Update 01_table_material.sql

criação da tabela material

Viniciusog commented on esquema_fisico/01_table_material.sql:
- Renomear PK_material_id para Pk_material
- Renomear material_anterior para material_anterior_id
- Renomear codigo_componente para componente_curricular_codigo

pinguim11 committed:
Update 01_table_material.sql

alteração de PK_material_id para Pk_material,
alteração de material_anterior para material_anterior_id,
alteração de codigo_componente para componente_curricular_codigo

pedro-salviano reviewed (approved):
LGTM!
